index.js

```diff
@@ -20,14 +20,17 @@ class MyCustomReporter {
     results.config = this._globalConfig
     results.endTime = Date.now()
     results._reporterOptions = this._options
-    const data = JSON.stringify(results)
+    let data = JSON.stringify(results)
     const {
       publicPath = process.cwd(),
       filename = 'jest_html_reporters.html',
+      hideIcon = false,
+      pageTitle = 'Report'
     } = this._options
     fs.existsSync(publicPath) === false && publicPath && mkdirs(publicPath)
     const filePath = path.resolve(publicPath, filename)
     const htmlTemplate = fs.readFileSync(localTemplatePath, 'utf-8')
+    data = JSON.stringify({ testResult: results, config: { pageTitle, hideIcon } });
```

Comment:
`results` have included all options (at line 23), seems needn't add it in here.

Other things look good for me. Due to respond too late I will merge this pr first, then do a small fix in here and let you know.

Thank you for contribution again.

Reply:
Thanks, @Hazyzh, 
It looks better now.